backend/functions/schemes/search.py

```diff
@@ -0,0 +1,79 @@
+"""
+url for local testing:
+http://127.0.0.1:5001/schemessg-v3-dev/asia-southeast1/schemes_search
+"""
+
+import json
+
+from fb_manager.firebaseManager import FirebaseManager
+from firebase_functions import https_fn
+from loguru import logger
+
+from ml_logic import PredictParams, SearchModel
+
+
+search_model = None
+
+
+def init_model():
+    """Initialises SearchModel class"""
+
+    global search_model
```

Comment:
generally not a good practice to use global variables because it will lead to testing and code readability issues, as it is not always clear where and how the global state is being modified. People typically use dependency injection or factory functions.

Can try to use factory function.